PukingPredator/Assets/Scripts/Consumable.cs

```diff
@@ -214,4 +223,27 @@ public void StartDecay()
         decayTimer.StartTimer(decayTime);
         decayTimer.onTimerComplete += Decay;
     }
+
+    private void ConfigureOutlines()
+    {
+        outline = gameObject.AddComponent<Outline>();
+        outline.OutlineWidth = OUTLINERADIUS;
+        outline.OutlineMode = Outline.Mode.OutlineVisible;
+    }
+
+    private void IsPlayerNear()
```

Comment:
would it be better to use the logic in Eating.cs to highlight the object that the player would eat if they clicked? I see that multiple objects can be highlighted if the player is near them